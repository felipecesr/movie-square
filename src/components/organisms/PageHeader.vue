<template>
  <header>
    <logo/>
  </header>
</template>

<script>
import Logo from '@atoms/Logo.vue';

export default {
  components: { Logo }
}
</script>

<style lang="scss" scoped>
header {
  padding-top: 3.3rem;
}
</style>
