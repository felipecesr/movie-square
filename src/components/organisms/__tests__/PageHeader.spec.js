import { shallowMount } from '@vue/test-utils';
import PageHeader from '../PageHeader.vue';

describe('PageHeader.vue', () => {
  it('renders correctly', () => {
    expect(shallowMount(PageHeader).element).toMatchSnapshot();
  });
});
