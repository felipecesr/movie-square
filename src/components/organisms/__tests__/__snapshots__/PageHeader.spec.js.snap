// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageHeader.vue renders correctly 1`] = `
<header>
  <logo-stub />
</header>
`;
